webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -201,8 +207,9 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
             lbl.text(o.searchLabel);
 
             //3 should be the minimum number of results to display per page
-            if(o.displayLength < 3)
+            if(o.displayLength < 3) {
                 o.displayLength = 3;
+			}
```

Comment:
Watch out for indentation issues. There are a lot of lines in this file where the braces are kinda misaligned.

Reply:
Oh I will fix those. Thank you!

Reply:
I think its just the diff making it look like that; in the actual file, they are aligned.

Reply:
I think its our local editors doing the formatting for us. I opened the file using VSCode and nothing was odd. But when I opened it using gedit, there were a lot of misaligned braces. I think the problem is being caused by the wrong number of tabs, so clearing the tabs before the curly brace, and entering them might help.

Reply:
I couldn't find a fast way to solve this issue, however, I did solve it manually. Thanks for the heads up. I didn't know that gedit would do that.